src/pages/explorer.js

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import Explorer from "cpal-components"
+
+import { Col, Row, Collapse } from "react-bootstrap"
+import Layout from "../components/layout"
+import SEO from "../components/seo"
+import Hero from "../components/hero"
+import { pages } from "../consts"
+// <Row noGutters className="">
+//   <Col
+//     className=""
+//     xs={{ offset: 0, span: 12 }}
+//     md={{ offset: 5, span: 7 }}
+//   ></Col>
+// </Row>
+//
+// <Layout id="faqs-page" activePageId={pages.EXPLORER.id}>
+//   <SEO title="Explorer" />
+//   <Explorer />
+// </Layout>
+
+const ExplorerPage = () => {
+  return (
+    <>
+      <SEO title="Explorer" />
+      <Explorer />
+    </>
```

Comment:
@amygroshek is the Explorer supposed to get the `StickyHeader` for a menu, and the footer elements? (could be a question for @james-minton) Without wrapping this in `<Layout activePageId={pages.EXPLORER.id}>` it won't get the footer elemenst, and the `StickyHeader` will appear, but when you open the menu Explorer won't be highlighted as active

Reply:
@hoshmn No, we don't want the parent site footer for this. It's not supposed to scroll. And I can't use your header because I need the explorer search bar in it. Please just merge so I can get working.